dev/PowerNotifications/PowerNotifications.idl

```diff
@@ -0,0 +1,115 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+// REVIEW: We're not doing doing any versioning within IDL/WinMDs, right?
```

Comment:
We are, and @MikeHillberg / @jevansaks  are building guidance for it. For now, you can leave this as is and we'll apply versioning markup prior to release.